packages/web-components/src/components/va-alert/test/va-alert.e2e.ts

```diff
@@ -11,7 +11,7 @@ describe('va-alert', () => {
     expect(element).toEqualHtml(`
       <va-alert class="hydrated">
         <mock:shadow-root>
-          <div class="alert info">
+          <div class="alert hide-icon info">
```

Comment:
Can we make sure to only add this class if `background-only` is truthy?

Reply:
Done. Good catch!